lib/@nodejs/http2.js

```diff
@@ -0,0 +1 @@
+module.exports = require('http2');
```

Comment:
I was under the impression we might want to do this in the resolver as opposed to creating this extra level of indirection. I'm not 100% the best approach but will follow up with something.

Reply:
The aliases are a finite list of one-offs; I don't mind putting it in the resolver, but I wanted to showcase how a *new* module would work (i just didn't have a new module ready to go)

Reply:
Definitely see what you're going for @ljharb, but I think doing this in the resolver is definitely the better approach... *even* with new modules. In the resolver, we should likely simply whitelist the legacy ones that should continue to work without `@nodejs/` and require all other core modules to have the `@nodejs/` included.

Reply:
@jasnell I’m not clear on what the benefit is of using the resolver for new modules, unless you’re suggesting a new way for core modules to work? My assumption is that `@nodejs/new_thing` would work identically to the way `fs` works - reading it off the filesystem.